themes/_core/js/checkout.js

```diff
@@ -41,25 +41,20 @@ function handleCheckoutStepChange() {
 
   let currentStepClass = 'js-current-step';
   let currentStepSelector = '.' + currentStepClass;
-  let stepsAfterPersonalInformation = $('#checkout-personal-information-step').nextAll();
 
-  $(currentStepSelector).prevAll().add(stepsAfterPersonalInformation).on(
+  $(currentStepSelector).prevAll().on(
     'click',
     (event) => {
-      let $nextStep = $(event.target).closest('.checkout-step');
-      if (!$nextStep.hasClass('-unreachable')) {
+      let $clickedStep = $(event.target).closest('.checkout-step');
```

Comment:
```suggestion
      const $clickedStep = $(event.target).closest('.checkout-step');
```

Reply:
Done! (it's funny it does change the built js..) It must be converted back to a `var`

Reply:
yep :)